external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/OracleIntegrationSuite.scala

```diff
@@ -48,7 +48,7 @@ import org.apache.spark.tags.DockerTest
  *  $ git clone https://github.com/oracle/docker-images.git
  *  // Head SHA: 3e352a22618070595f823977a0fd1a3a8071a83c
```

Comment:
Oh, here is SHA, @sarutak . In this case, we need to update this SHA too, @sarutak .

Reply:
Ah, I didn't care about it. I'll fix it.

Reply:
The SHA is `3f422c4a35b423dfcdbcc57a84f01db6c82eb6c1`